packages/colors/src/design-system-colors.test.ts

```diff
@@ -0,0 +1,73 @@
+import { tint, shade } from 'polished';
+import { pipe } from 'ramda';
+import Colors, { baseColors } from './design-system-colors';
```

Comment:
<img width="359" alt="with libs" src="https://user-images.githubusercontent.com/24607370/72114491-f71e3980-3343-11ea-8dd1-c3238bf1cecb.png">
Decided to use scripts from Michał Jawinski only in tests because adding polished and ramda increased size of package 6 times (when gzipped)